src/unity/python/turicreate/toolkits/recommender/util.py

```diff
@@ -569,13 +569,12 @@ def _get(self, field):
         Examples
         --------
         >>> data = turicreate.SFrame({'user_id': ["0", "0", "0", "1", "1", "2", "2", "2"],
-        ...                         'item_id': ["a", "b", "c", "a", "b", "b", "c", "d"],
-        ...                         'rating': [1, 3, 2, 5, 4, 1, 4, 3]})
-        >>> from turicreate.recommender
-        >>> m = factorization_recommender.create(data, "user_id", "item_id", "rating")
-        >>> d = m.get("coefficients")
+                                      'item_id': ["a", "b", "c", "a", "b", "b", "c", "d"],
+                                      'rating': [1, 3, 2, 5, 4, 1, 4, 3]})
+        >>> m = turicreate.factorization_recommender.create(data, "user_id", "item_id", "rating")
+        >>> d = m._get("coefficients")
```

Comment:
Please change this doc string to 

```
m.coefficients
```

Reply:
I don't think that makes sense. This is a docstring for `_get(...)`.

Given that there is a better way to get coefficients, perhaps it would be best to get something other than coefficients using `_get`. What do you think? Any idea what would make sense to get?

Reply:
Oh. Okay. Never mind then.